### ISSUE TITLE
[Bug]: Windows Build instructions and packages are missing

### ISSUE THREAD

**Issue by richlysakowski** [sqlitebrowser/sqlitebrowser#3191]
### What did you do?

The home page for the build instructions is broken.  I wanted to try to build it from scratch.   The instructions look phenomenal, but most of the links to the tools are not working.  

I already have Visual Studio 2022 Community Edition and GithubDesktop installed.  I need the rest of the tools (updated links and instructions would be nice.) 

When I click on the link for each required product in the Windows Build instructions, they all point to the same file (CMake, SQLite, Nullsoft Scriptable Install System, OpenSSL, Qt, SQLite), NOT to the individual website and download for each tool.  

TO REPRODUCE THIS RESULT: 

Go to "Setting up a Win64 development environment for DB4S" at https://github.com/sqlitebrowser/sqlitebrowser/wiki/Setting-up-a-Win64-development-environment-for-DB4S 

Each link below points to THE SAME FILE named "AppSetup.zip" (at https://github.com/bonkonauts/cors-anywhere/releases/download/v0.1.9/AppSetup.zip ), **RATHER than pointing to each tool's website**.  

#####################
[CMake 3.6.1 or later](https://github.com/bonkonauts/cors-anywhere/releases/download/v0.1.9/AppSetup.zip)
Under the "Binary distributions" heading, choose the Windows Installer
Make sure you get version 3.6.1 or later. Earlier versions can introduce security problems.

[GitHub Desktop for Windows](https://desktop.github.com/)

GitHub Desktop for Windows 7 or later

[Microsoft Visual Studio Community 2013 Edition Update 5](https://www.visualstudio.com/downloads/download-visual-studio-vs) Click the "Visual Studio 2013" option 1/2 way down the left side → "Community 2013" → ISO
The latest [Microsoft Visual C++ Redistributable Packages for Visual Studio 2013 (x64)](https://support.microsoft.com/en-us/help/3138367/update-for-visual-c-2013-and-visual-c-redistributable-package)
Choose the "vcredist_x64.exe" option

[Nullsoft Scriptable Install System](https://github.com/bonkonauts/cors-anywhere/releases/download/v0.1.9/AppSetup.zip)
Version 3.0rc1 or later

[OpenSSL](https://github.com/bonkonauts/cors-anywhere/releases/download/v0.1.9/AppSetup.zip)
From the first link (Shining Light Productions) → Win64 OpenSSL (use latest version)

[Qt](https://github.com/bonkonauts/cors-anywhere/releases/download/v0.1.9/AppSetup.zip)
Choose the Qt Online Installer for Windows

[SQLite](https://github.com/bonkonauts/cors-anywhere/releases/download/v0.1.9/AppSetup.zip)
Under the "Source Code" heading, download the "sqlite-amalgamation" .zip file
Version 3.12.2 was used for this guide, but other versions will probably work without issue too

### What did you expect to see?

Working links to the required tools.  

(Updated instructions would be nice.) 

### What did you see instead?

The same file "AppSetup.zip" instead of the different downloads required to do the build process.  

### DB4S Version

3.12.2

### What OS are you seeing the problem on?

Windows

### OS version

Windows 10

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by FriedrichFroebel** [comment]
It seems like the same guy that replaced these links 10 days ago reverted the changes back about 10 minutes after you reported it: https://github.com/sqlitebrowser/sqlitebrowser/wiki/Setting-up-a-Win64-development-environment-for-DB4S/_history

**Comment by chrisjlocke** [comment]
I downloaded a lot of these programs the other day to try the build process.
I could download the files (after a bit of hunting) but couldn't get the actual build to work.  Windows just reported an error.  There is an issue floating around here with my results and whimpering in the corner.
Would be keen to 'bash heads' if possible so I could learn a bit on where I went wrong, as while I've got a Linux box too, I'm predominantly a Windows user. I sort of know what I'm doing, but not enough in this area....

**Comment by justinclift** [comment]
Thanks for the letting us know @richlysakowski.  I've just reported the (malicious?) change to GitHub, so they'll hopefully figure out that aspect of things.

For the instructions themselves... yeah, they're very old.  That being said, the process itself hasn't changed much.  I don't know if anyone's been building for Windows on Win 10 or later (the ancient nightly build VM still runs Win 7 :wink:).

My suspicion is that for building on Win 10/11, you'd *probably* need to:

1. Use MSVC 2019
2. For the Qt download choose Qt Qt 5.15.x, then the MSVC 2019 sub-option (so it matches with 1. above)

One thing that definitely needs changing, is that we no longer use NSIS for the automatic packaging step.  Instead we moved to using [WiX Toolset](https://wixtoolset.org).  The latest version should be fine.

Some other data points that are probably worth mentioning:

* For CMake, the very latest current release works fine for our macOS builds.  So it's probably fine on Windows too.
* For OpenSSL, choose the 1.1.1 series - and probably not the *Light* ones - as that's known to work.

Does that help? :smile:
